jobs/issues_status.rb

```diff
@@ -10,9 +10,9 @@
 	opened_series = [[],[]]
 	closed_series = [[],[]]
 	issues_by_period = backend.issue_count_by_status(
-		:orgas=>(ENV['ORGAS'].split(',') if ENV['ORGAS']), 
-		:repos=>(ENV['REPOS'].split(',') if ENV['REPOS']),
-		:since=>ENV['SINCE']
+		orgs: (ENV['ORGS'].split(',') if ENV['ORGS']),
```

Comment:
Indent the first parameter one step more than the previous line.<br>Tab detected.